src/features.js

```diff
@@ -0,0 +1,99 @@
+const dataArr = [
+  {
+    name: 'Black Sherif',
+    img: './images/black_sherif_artise.jpg',
+    profession: 'Afro beat, Singer and Rapper',
+    description:
+      "Mohammed Ismail Sherif Kwaku Frimpong professionally known as Black Sherif, is a Ghanaian singer and rapper. He gained popularity in 2021 with his song 'First Sermon' in May 2021. This was followed up with 'Second Sermon' in July. ",
```

Comment:
- [ ] For these feature speakers kindly consider using grey chess as shown on the design provided by Microverse instead of a black one
![Screenshot from 2023-02-10 04-59-46](https://user-images.githubusercontent.com/97834160/217989788-3b35ccac-3969-442c-8f1a-69d8c933c01b.png)
![Screenshot from 2023-02-10 04-59-33](https://user-images.githubusercontent.com/97834160/217989818-ace0c2be-0c58-4124-8e19-7b451a444100.png)